tests/core/pyspec/eth2spec/test/altair/block_processing/test_process_sync_committee.py

```diff
@@ -68,8 +68,6 @@ def test_invalid_signature_missing_participant(spec, state):
     rng = random.Random(2020)
     random_participant = rng.choice(committee)
 
-    yield 'pre', state
-
```

Comment:
these seem fine but i do wonder if there is any concern about accidentally mutating the pre-state before getting to the inner yield in `run_sync_committee_processing`